### ISSUE TITLE
[releasenotes-builder] NPE on Travis CI

### ISSUE THREAD

**Issue by romani** [checkstyle/contribution#131]
checkstyle Travis CI is red.

reverted: https://github.com/checkstyle/contribution/pull/130 now release notes is fine - https://travis-ci.org/checkstyle/checkstyle/jobs/179613449

Not relaunched build:
https://travis-ci.org/checkstyle/checkstyle/jobs/179594884
```
Exception in thread "main" java.lang.NullPointerException
	at java.util.Objects.requireNonNull(Objects.java:203)
	at java.util.Optional.<init>(Optional.java:96)
	at java.util.Optional.of(Optional.java:108)
	at com.github.checkstyle.NotesBuilder.getActualRefObjectId(NotesBuilder.java:226)
	at com.github.checkstyle.NotesBuilder.getCommitsBetweenReferences(NotesBuilder.java:152)
	at com.github.checkstyle.NotesBuilder.buildResult(NotesBuilder.java:84)
	at com.github.checkstyle.Main.runNotesBuilder(Main.java:147)
	at com.github.checkstyle.Main.main(Main.java:93)
```

**Comment by romani** [comment]
@MEZk , please address asap, your commit was reverted.

**Comment by romani** [comment]
fixed by #132
